src/ape_ethereum/_converters.py

```diff
@@ -37,3 +38,15 @@ def convert(self, value: str) -> int:
         value, unit = value.split(" ")
 
         return int(Decimal(value) * ETHER_UNITS[unit.lower()])
+
+
+class TimestampConversions(ConverterAPI):
+    """Converts seconds to date and time string value"""
+
+    def is_convertible(self, value: int) -> bool:
+        if not isinstance(value, int):
+            return False
+        return True
+
+    def convert(self, value: int) -> str:
+        return datetime.fromtimestamp(value).strftime('%m-%d-%Y %H:%M:%S')
```

Comment:
I was picturing the reverse direction, since timestamps in EVM are always in seconds, but for users they can write their timestamps as ISO strings for arguments that take timestamps:

```py
chain.pending_timestamp = "02-01-2022 12:00:00"
# Converts to an integer value inside of `pending_timestamp.setter`
```

Reply:
Note that you actually have to add this `convert(new_value, int)` statement to the chain manager [here](https://github.com/ApeWorX/ape/blob/main/src/ape/managers/chain.py#L373)

This means you have to inject the `converters` Conversion Manager inside of the `chain` Chain Manager when it is created [here](https://github.com/ApeWorX/ape/blob/main/src/ape/__init__.py#L50)

(See [here](https://github.com/ApeWorX/ape/blob/main/src/ape/__init__.py#L57) for an example of this injection)

Reply:
I will reverse these

Reply:
Would it make sense to move `TimestampConversions` to the core-ape converters?

Reply:
Should I move this from ape_ethereum to managers/converters.py?

We went over this and we were looking at the WeiConverter class, but converters looks like a better place for this

Reply:
@unparalleled-js I didn't reload the page so I wrote the same question, but I believe we should

Reply:
yes, see my other comment, please mv to `src/ape/managers/converters.py`